### PR TITLE
Ocata Release

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
[Convert osl-openstack to support Ocata](https://trello.com/c/GC9GscBL/563-convert-osl-openstack-to-support-ocata)

**Comment by ramereth** [comment]
[Enable OpenStack Orchestration service (heat)](https://trello.com/c/Gm4cKBtQ/567-enable-openstack-orchestration-service-heat)

**Comment by ramereth** [comment]
@osuosl-cookbooks/chefs I believe this is finally ready for a few. Any new changes should be minor

**Comment by ramereth** [comment]
@TravisWhitehead I believe I've resolved those issues. Seems there was an update in one of the python libraries that just happened causing an issue with the openstack binary. I also fixed the nova placement test so that it should work every time.

I did notice that the ``compute-centos-7`` and ``allinone-centos-7`` may need an extra run of verify or converge to get it to pass.

File: spec/default_spec.rb [hunk 1]
```diff
@@ -113,7 +106,14 @@
   it do
     expect(chef_run).to install_python_package('python-openstackclient')
       .with(
-        version: '3.11.0',
+        version: '3.14.0',
+        # virtualenv: '/opt/osc'
```
Comment (TravisWhitehead) [inline spec/default_spec.rb]: Why comment this out?

Comment (ramereth) [reply spec/default_spec.rb]: ChefSpec doesn't support including it. See what happens when you remove the comment.

**Comment by ramereth** [comment]
!bump major openstack_ocata

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v5.0.0.  Have a nice day!

**Pull request merged**